content/company-info-and-process/communication/content_guidelines/actionable_language.md

```diff
@@ -599,3 +599,17 @@ This is particularly important for helper messages on input fields.
 
 </div>
 </div>
+
+### Writing about use cases
+
+When prompting users about or writing about our 5 company uses cases, we can't assume that users are familiar with our terminology: a developer does not necessarily have in mind that they are using Sourcegraph for “Code Health”.
+
+Instead we can use this more intuitive (but less accurate) language:
+
+| Sourcegraph use case                                                                                             | "You are using Sourcegraph to..."     |
+| ---------------------------------------------------------------------------------------------------------------- | ------------------------------------- |
+| [Developer Onboarding](../../../strategy-goals/strategy/use-cases/dev-onboarding.md)                             | Understand a new part of the codebase |
+| [Code Health](../../../strategy-goals/strategy/use-cases/code-health.md)                                         | Track and improve codebase quality    |
```

Comment:
```suggestion
| [Code Health](../../../strategy-goals/strategy/use-cases/code-health.md)                                         | Track and improve codebase quality    |
```

@malomarrec what if we removed "Track and" and just said "improve"? Which includes tracking, but is less likely to cause the overmatching problem on code insights use cases all getting bucketed into this single bucket that I described here? Sorry for not thinking of this sooner. https://docs.google.com/document/d/1_ClgFMY1aak3xG533j1KQYmIbx6u_QUnB8GL_GQWocM/edit?disco=AAAAVP9l0p0